cognite/neat/core/workflow/manager.py

```diff
@@ -14,12 +14,20 @@
 
 
 class WorkflowManager:
+    """Workflow manager is responsible for loading, saving and managing workflows
+    client: CogniteClient
+    registry_storage_type: str = "file"
+    workflows_storage_path: Path = Path("workflows")
+    rules_storage_path: Path = Path("rules")
+    data_set_id: int = None,
+    """
+
     def __init__(
         self,
-        client: CogniteClient,
-        registry_storage_type: str,
-        workflows_storage_path: Path,
-        rules_storage_path: Path,
+        client: CogniteClient = None,
+        registry_storage_type: str = "file",
+        workflows_storage_path: str = "workflows",
+        rules_storage_path: str = "rules",
```

Comment:
Path objects? That is what is sent in through the `NeatApp`

```suggestion
        workflows_storage_path: Path = Path("workflows"),
        rules_storage_path: Path = Path("rules"),
```

Reply:
Originally it was the same as in suggested solution but linter is complaining by stating that function should not be used  for generating default values . Not sure how to solve it in a good way.

Reply:
Skip linter on that line maybe :)